### PR TITLE
[TASK-004] - Create and read data from "materias.txt" file.

### PR DESCRIPTION
**Pull request by carlosfigueroa97**
Please review my code bro !. I didn't test my code but I trust me jaja.
Regards.

### CODE REVIEW

File: connection.py [hunk 1]
```diff
@@ -174,18 +174,31 @@ def drop_tables(db):
...
-            n_alumno = 
+            
             return 
         elif(table == 'Materia'):
-
+            n_subject = get_randon_data_materias()
         else:
-
-    except:
+            print('blablabla')
```
Comment (Colima-97) [inline connection.py]:
```suggestion
            pass
```

File: connection.py [hunk 2]
```diff
@@ -174,18 +174,31 @@ def drop_tables(db):
...
     try:
         cursor = db.cursor()
 
         if(table == 'Alumnos'):
-            n_alumno = 
+            
             return 
         elif(table == 'Materia'):
-
+            n_subject = get_randon_data_materias()
```
Comment (Colima-97) [inline connection.py]:
```suggestion
            n_subject = get_randon_data_materias()
            return n_subject
```

File: connection.py [hunk 3]
```diff
@@ -174,18 +174,31 @@ def drop_tables(db):
     else:
         print("Tablas borradas!")
 
+def read_data_materias():
+    listMaterias = []
+    materias = open('materias.txt', 'r')
+    for i in materias.readlines():
+        if(i != None or i != ""):
+            listMaterias.append(i)
```
Comment (Colima-97) [inline connection.py]:
```suggestion
            i = i.rstrip("\n")
            listMaterias.append(i)
```

File: connection.py [hunk 4]
```diff
@@ -174,18 +174,31 @@ def drop_tables(db):
...
+    return listMaterias
+
+def get_randon_data_materias():
+    response = read_data_materias()
+    matter = r.choice(response)
```
Comment (Colima-97) [inline connection.py]:
```suggestion
def get_random_data_materias(listMaterias):
    matter = r.choice(listMaterias)
```

File: connection.py [hunk 5]
```diff
@@ -174,18 +174,33 @@ def drop_tables(db):
     else:
         print("Tablas borradas!")
 
+def read_data_materias():
```
Comment (Colima-97) [inline connection.py:180]: Bro, I have a doubt...

If we call the method `read_data_materias()` every time we need to choice a *subject*, we'd fill the list like a million times!

So, I was thinking about declaring the list at the very begining, and calling the method `read_data_materias()` just before the first `for` loop starts.

If we do that, then, the `get_random_data_materias()` won't be needed, it'd be better to just put `subject = r.choice(listMaterias)`

What do you think?

**Review by Colima-97 — CHANGES_REQUESTED** [review]
Check the comments bro

File: connection.py (hunk 5 above)
Comment (carlosfigueroa97) [reply connection.py:180]: I agree with you!

File: connection.py [hunk 6]
```diff
@@ -196,8 +193,8 @@ def r_data(db, table):
             
             return 
         elif(table == 'Materia'):
-            n_subject = get_randon_data_materias()
-            return n_subject
+            n_subject = r.choice(list_materias)
+            # next logic
```
Comment (Colima-97) [inline connection.py]:
```suggestion
            return n_subject
```

**Review by Colima-97 — APPROVED** [review]
Nice job bro

**Pull request merged**